src/main/java/org/jenkinsci/plugins/verifystatus/VerificationsPublisher.java

```diff
@@ -173,11 +173,19 @@ public boolean perform(AbstractBuild build, Launcher launcher,
             gerritConfig.getGerritHttpUserName(),
             gerritConfig.getGerritHttpPassword());
     GerritApi gerritApi = gerritRestApiFactory.create(authData);
+    int changeNumber;
+    int patchSetNumber;
+    try {
+      changeNumber = Integer.parseInt(
```

Comment:
Ahh, I think you only get this if you trigger from Jenkins (via gerrit trigger query).  I only tested triggering from Gerrit so I never noticed.  Nice fix.